aiosmtpd/handlers.py

```diff
@@ -148,13 +148,34 @@ def process_message(self, peer, mailfrom, rcpttos, data, **kws):
         message['X-Peer'] = str(peer)
         message['X-MailFrom'] = mailfrom
         message['X-RcptTos'] = COMMASPACE.join(rcpttos)
+
+        return message
+
+    def process_message(self, peer, mailfrom, rcpttos, data, **kws):
+        message = self.prepare_message(peer, mailfrom, rcpttos, data, **kws)
```

Comment:
I'm curious as to why you split this out into `prepare_message()`.  I don't necessarily think it's a bad thing to do, just curious as to your thinking here.